src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2448,11 +2469,6 @@ public override BoundNode VisitIncrementOperator(BoundIncrementOperator node)
                     resultOfIncrementType = GetTypeOrReturnTypeWithAdjustedNullableAnnotations(incrementOperator);
                 }
 
-                if (node.ResultConversion.IsUserDefined && (object)node.ResultConversion.Method != null && node.ResultConversion.Method.ParameterCount == 1)
```

Comment:
>if (node.ResultConversion.IsUserDefined && (object)node.ResultConversion.Method != null && node.ResultConversion.Method.ParameterCount == 1) [](start = 16, length = 140)

Is the assumption that InferResultNullability call below will perform this check?

Reply:
Yes.

---
In reply to: [183865871](https://github.com/dotnet/roslyn/pull/26356#discussion_r183865871) [](ancestors = 183865871)